src/Microsoft.VisualStudio.SDK.Analyzers.CodeFixes/VSSDK001DeriveFromAsyncPackageCodeFix.cs

```diff
@@ -82,7 +82,7 @@ private async Task<Document> ConvertToAsyncPackageAsync(CodeFixContext context,
             }
 
             // Make it easier to track nodes across changes.
-            var nodesToTrack = new List<SyntaxNode>
+            var nodesToTrack = new List<SyntaxNode?>
```

Comment:
I agree. In fact a couple statements down you see we remove all the null elements. So I guess this was just to allow us to add all 4 syntaxes that might apply with simpler syntax. I was tempted to clean this up, but since adding the annotation that simply identified what we were already doing solved the build warnings, I decided to not clean it up further as part of this PR.